src/osp/Resource/Package.h

```diff
@@ -217,9 +220,14 @@ DependRes<TYPE_T> Package::get(std::string_view path)
 }
 
 template<class TYPE_T>
-void Package::clear()
+void Package::clear() noexcept
 {
-    m_groups.erase(entt::type_info<TYPE_T>::id());
+    const uint32_t resId = resource_id::type<TYPE_T>;
+    //m_groups.erase(entt::type_info<TYPE_T>::id());
+    if (m_groups.size() > resId)
+    {
+        m_groups[resId] = entt::any{}; // Destruct GroupType
```

Comment:
Why not resize to one smaller?

Reply:
There is other groups with a later index still stored in there. Only a single type is being removed.